views/pageTemplate.njk

```diff
@@ -5,6 +5,7 @@
         <title> Kainos Career Portal</title>
         <link rel="stylesheet" href="style.css"/>
         <link rel="icon" href="images/Green_Circle_Favicon.png" type="image/png" id="kainosFavicon">
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```

Comment:
why do we need bootstrap?

Reply:
I use it for the login form, is that okay?